app/routes/create.js

```diff
@@ -14,8 +14,10 @@ export default Route.extend(AuthenticatedRouteMixin, EventWizardMixin, {
         copyright           : this.store.createRecord('event-copyright'),
         stripeAuthorization : this.store.createRecord('stripe-authorization')
       }),
-      module : await this.get('store').queryRecord('module', {}),
-      types  : await this.store.query('event-type', {
+      module         : await this.get('store').queryRecord('module', {}),
+      permissions    : await this.store.findRecord('user-permission', 2),
```

Comment:
what is this? @uds5501

Reply:
@CosmicCoder96 I am fetching system-permissions here.
As for using `findRecord`, to pin point the `create_event` permissions, I was initially using the following :

```js
permissions    : await this.store.query('user-permission', {
        filter : [{
          name : 'name',
          op   : 'eq',
          val  : 'create_event'
        }]
      }),
```
but it still gave me permission for both `create_event` and `publish_event`

Reply:
I don't understand. The scope of this issue is: disable the button if user is unverified. You don't need to make this extra network call.

Reply:
@CosmicCoder96 I was just considering the case where the system permissions allow an unverified user to create an event :sweat_smile:

Reply:
@uds5501 Not now -- I am really trying to restrict the number of network requests sent out.

Reply:
@CosmicCoder96 Understood!